### ISSUE TITLE
bug: filetypes as `SpooledTemporaryFile` for web-app api

### ISSUE THREAD

**Issue by LaverdeS** [Unstructured-IO/pipeline-general#42]
In our efforts to refine the Dockerfile in [this PR](https://github.com/Unstructured-IO/pipeline-general/tree/docs/Dockerfile-update) we found that both the web app successfully parse some document types but for `pptx` it raises: `AttributeError: 'SpooledTemporaryFile' object has no attribute 'seekable'` (same should happen for main branch but has not been tested). The same occurs for the container parsing `.docx` files.

**How to reproduce:**
- For the image/container: `make docker-build` && `make docker-start-api`
- For the web app: `make run-web-app`
- `curl -X 'POST' 'http://localhost:8000/general/v0.0.4/general' -F 'files=@sample-docs/fake.doc'` (OR using `@sample-docs/fake-power-point.ppt` or `@sample-docs/fake.docx`.

**Definition on done:**
- Successfully curl POST the `.pptx`, `.docx`, `.html` and `.eml` documents in the [sample-docs](https://github.com/Unstructured-IO/pipeline-general/tree/docs/Dockerfile-update/sample-docs) folder (web app and docker container. The output should be a bunch of Unstructured elements.

**Comment by cragwolfe** [comment]
In the same container, have you confirmed if you start a terminal and manually do

```
from unstructured.partition.auto import partition
elements = partition(filename="the-file.docx")
```
that it works? i'm guessing it would, but would be good to confirm that.

Withouth the stack trace, my guess it is a libmagic issue which implies we should not avoid using libmagic in the context of the api in https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/file_utils/filetype.py , but rely on the `file_content_type` arg that gets passed into `pipeline_api` https://github.com/Unstructured-IO/pipeline-template/blob/main/%7B%7B%20cookiecutter.repo_name%20%7D%7D/pipeline-notebooks/pipeline-process-file.ipynb . Of course if that's all true, we'll need a function to go from `file_content_type` to `FileType` :-|

Also, please paste the stack trace somewhere (could be a link to a github gist) 🙏

**Comment by LaverdeS** [comment]
Inside the container `docx` and `pptx` work fine. I also tried the other filetypes and got some new errors though:
- `.ppt`: the gist is [here](https://gist.github.com/LaverdeS/e3e48ce49bdb051a781d332d98344913).

**Comment by LaverdeS** [comment]
Fixed in #44
